packages/scandipwa/public/index.html

```diff
@@ -3,17 +3,26 @@
 <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no, viewport-fit=cover">
-    
+
     <!-- Muli font import from Abode -->
     <link rel="stylesheet" href="https://use.typekit.net/gbk7rfi.css">
 
     <!-- Default Meta -->
     <title>ScandiPWA</title>
     <meta name="theme-color" content="#ffffff" />
     <meta name="description" content="Web site created using create-scandipwa-app" />
-    
+
     <!-- Default content-configurations -->
     <script>
+        (function() {
+                if (typeof globalThis === 'object') return;
+                Object.prototype.__defineGetter__('__magic__', function() {
+                    return this;
+                });
+                __magic__.globalThis = __magic__;
+                delete Object.prototype.__magic__;
+            }());
```

Comment:
Is it possible to separate this into some other file? In order to keep the HTML template only for HTML.

Reply:
It's possible, I didn't do this because solution was copy-pasted from our `index.php`: https://github.com/scandipwa/scandipwa/blob/master/packages/m2-theme/public/index.php#L16

Now extracted into separate file.